tpot/tpot.py

```diff
@@ -790,6 +792,51 @@ def _gradient_boosting(self, input_df, learning_rate, max_features, min_weight):
             learning_rate=learning_rate, n_estimators=500,
             max_features=max_features, random_state=42, min_weight_fraction_leaf=min_weight)
 
+    def _mdr(self, input_df, tie_break, default_label, num_features_to_combined = 2, score = None):
+        """Fits the Multi-dimensionality Reduction Classifier/Feature Constructor
+
+        Parameters
+        ----------
+        input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+            Input DataFrame for fitting the MDR feature constructor
+        tie_break: int
+            One of the class values (currently categorical) that determines which class to assign to the combined feature instance, in case class ratio among 
+        possible classes of that feature instance vector is the same as the class ratio in the general population.
+            Maximum number of features to use (proportion of total features)
+        default_label: int
```

Comment:
For both `tie_break` and `default_label`, allow TPOT to specify these parameters as you already have.

However, make sure that TPOT is choosing one of the existing classes in the data set. This can be accomplished by, e.g.,

```
all_classes = input_df['class'].unique() # make sure this is a consistently sorted list
tie_break_choice = all_classes[tie_break % len(all_classes)]
default_label_choice = all_classes[default_label % len(all_classes)]
```